symphony/lib/toolkit/class.pagemanager.php

```diff
@@ -301,11 +301,11 @@ public static function editPageChildren($page_id = null, $page_path = null)
             );
 
             if (!PageManager::createPageFiles($page_path, $child['handle'], $child['path'], $child['handle'])) {
-                $success = false;
+                return false;
```

Comment:
Note this changes the current behaviour. As this is in a loop, previously some children may of been updated, whereas now, as soon as one fails, the execution will stop (even if some previously worked).